test/distributed/org/apache/cassandra/distributed/test/FailingRepairTest.java

```diff
@@ -150,6 +150,8 @@ public static void setupCluster() throws IOException
             if (throwable.getClass().toString().contains("InstanceShutdown") || // can't check instanceof as it is thrown by a different classloader
                 throwable.getMessage() != null && throwable.getMessage().contains("Parent repair session with id"))
                 return true;
+            if (throwable.getMessage().contains("Corrupted: Corrupted"))
```

Comment:
what i don't get, why is this passing in trunk and been stable there, but cep-15-accord is unstable?  What is unique about cep-15-accord that makes this unstable?

Reply:
the fact this error is making it to this code path means we are not handling the error, and we have been since 4.0... why is cep-15-accord not handling this error anymore?

Reply:
I found the root cause.  CASSANDRA-20069 removed the error handling that this test depends on, so we need to revert that to get the test passing, not telling the test to ignore this exception.